mindsdb/integrations/handlers/oracle_handler/tests/test_oracle_handler.py

```diff
@@ -6,14 +6,8 @@
 class OracleHandlerTest(unittest.TestCase):
     @classmethod
     def setUpClass(cls):
```

Comment:
We never execute this tests and I think they are outdated, let's remove the whole dir